### PR TITLE
[Rule Tuning] NTDS or SAM Database File Copied

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Adds wildcards on some process.args conditions to not miss executions that use quotes (affecting how process.args are parsed)

### CODE REVIEW

**Review by imays11 — APPROVED** [review]

File: rules/windows/credential_access_copy_ntds_sam_volshadowcp_cmdline.toml
Comment (Aegrah) [inline rules/windows/credential_access_copy_ntds_sam_volshadowcp_cmdline.toml:41]: Might be good to have a second thought with regards to the wildcards prior to the copy statements, might be resource intensive. Not sure how often a copy command is inserted in the middle of a command. Will leave that up to you, if you have good reasons to do so, LGTM!

**Review by Aegrah — APPROVED** [review]

**Pull request closed without merging**
